ui/lib/networks.js

```diff
@@ -0,0 +1,36 @@
+import {
+  MAINNET_CHAIN_ID,
+  ROPSTEN_CHAIN_ID,
+  RINKEBY_CHAIN_ID,
+  GOERLI_CHAIN_ID,
+  KOVAN_CHAIN_ID,
+  ROPSTEN_NETWORK_ID,
+  RINKEBY_NETWORK_ID,
+  GOERLI_NETWORK_ID,
+  KOVAN_NETWORK_ID,
+} from '../../shared/constants/network';
+
+const TEST_CHAINS = [
+  ROPSTEN_CHAIN_ID,
+  RINKEBY_CHAIN_ID,
+  GOERLI_CHAIN_ID,
+  KOVAN_CHAIN_ID,
+];
+const TEST_NETWORKS = [
+  ROPSTEN_NETWORK_ID,
+  RINKEBY_NETWORK_ID,
+  GOERLI_NETWORK_ID,
+  KOVAN_NETWORK_ID,
+];
+
+export function isTestChain(chainId) {
+  return TEST_CHAINS.includes(chainId);
+}
+
+export function isTestNetwork(networkId) {
+  return TEST_NETWORKS.includes(networkId);
+}
+
+export function isMainNetChain(chainId) {
```

Comment:
Nit: "Mainnet" is one word:
```suggestion
export function isMainetChain(chainId) {
```

Reply:
Also, this function already exists as a selector: https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/selectors/selectors.js#L282

It might make sense to re-use that, and add a `getIsTestnet` selector for the testnet case, rather than introducing this utility module. The only useful constant defined here is `TEST_CHAINS`, and that could be added to the shared network constants module.